src/Application/AnalyzeCommand.php

```diff
@@ -58,6 +61,8 @@ protected function configure(): void
         $this->addArgument('paths', InputArgument::IS_ARRAY | InputArgument::OPTIONAL, 'Directories or files to analyze');
         $this->addOption('debug', null, InputOption::VALUE_NONE, 'Enable debug mode');
         $this->addOption('xdebug', null, InputOption::VALUE_NONE, 'Enable xdebug mode');
+        $this->addOption('generate-baseline', null, InputOption::VALUE_NONE, 'Path to a file where the baseline should be saved');
+        $this->addOption('allow-empty-baseline', null, InputOption::VALUE_OPTIONAL, 'Allow to generate PHPStan empty baseline', false);
```

Comment:
Not sure if we need this option to be honest. If you want a baseline, and there are no errors, then you get an empty baseline.

Reply:
you're right. I had removed allow-empty-baseline.